windows/install_vcpython.ps1

```diff
@@ -6,12 +6,18 @@ $ErrorActionPreference = 'Stop'
 [Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12
 
 
-$vcpmsi = 'https://download.microsoft.com/download/7/9/6/796EF2E4-801B-4FC4-AB28-B59FBF6D907B/VCForPython27.msi'
+$vcpmsi = 'https://s3.amazonaws.com/dd-agent-omnibus/VCForPython27.msi'
 
 Write-host -ForegroundColor Green Downloading VC++ for Python
 $out = "$($PSScriptRoot)\vcp.msi"
 
 (New-Object System.Net.WebClient).DownloadFile($vcpmsi, $out)
+
+if ((Get-FileHash -Algorithm MD5 $out).Hash -ne "4E6342923A8153A94D44FF7307FCDD1F")
```

Comment:
Please switch to SHA256 or something else. MD5 is quite broken.

Reply:
It could happen that the file is changed in such a way that it produces the same MD5 (that's relatively easy to do).